easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-foss-2018b.eb

```diff
@@ -0,0 +1,73 @@
+# easybuild easyconfig
+#
+# Graphviz has moved to gitlab and it is no longer possible to get tagged
+# versions.  There is an open ticket this issue.
+
+easyblock = 'ConfigureMake'
+
+name = 'Graphviz'
+version = '2.40.1'
+
+homepage = 'http://www.graphviz.org/'
+description = """Graphviz is open source graph visualization software. Graph
+ visualization is a way of representing structural information as diagrams of
+ abstract graphs and networks. It has important applications in networking,
+ bioinformatics,  software engineering, database and web design, machine
+ learning, and in visual interfaces for other technical domains."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://graphviz.gitlab.io/pub/graphviz/stable/SOURCES/']
+sources = ['graphviz.tar.gz']
```

Comment:
This is just... stupid. At the very least, we should rename this to include a version, for example via:

```python
sources = [{'download_filename': 'graphviz.tar.gz', 'filename': SOURCELOWER_TAR_GZ}]
```

@fizwit Can you refer me to the open issue?

Reply:
@fizwit Ping on the issue that discusses the lack of properly versioned source tarballs?

Reply:
see https://gitlab.com/graphviz/graphviz/issues/1371